fastapi_utils/tasks.py

```diff
@@ -10,16 +10,25 @@
 
 NoArgsNoReturnFuncT = Callable[[], None]
 NoArgsNoReturnAsyncFuncT = Callable[[], Coroutine[Any, Any, None]]
+NoArgsNoReturnAnyFuncT = NoArgsNoReturnAsyncFuncT | NoArgsNoReturnFuncT
```

Comment:
This does not support older versions of python (as we support every version FastAPI supports)

Reply:
Well then maybe Union from typing? I see ` | None` in exsting code, do they also need to be fixed?

Reply:
Curious, I've noticed these as well, but the CI is particularly not happy here. Fix this for now, and I'll test with all Python versions locally and make whatever changes necessary